File: backend/app/src/models/persons/WarehouseWorker.py
from .Worker import Worker


class WarehouseWorker(Worker):
    def __init__(self, name, employed = True, working = False, production: int = 500):
        super().__init__(name=name, employed=employed, field_of_work="WarehouseWorker", working=working, production=production)

    # def fetch_population_state(self) -> dict:
    #     population_state: dict = super().fetch_population_state()
    #     population_state.update({

    #     })
        
    #     return population_state

File: backend_refactoring/CSharpApi/Models/BluePrints/Beings/WarehouseWorker.cs
namespace CSharpApi.Models.BluePrints.Beings
{
    public class WarehouseWorker : Worker
    {
        public WarehouseWorker(string workerName, bool employed = true, string profession = "worker", bool alive = true, string fieldOfWork = "warehouseWorker", bool working = false, int production = 500) : base(workerName, employed, profession, alive, fieldOfWork, working, production)
        {
        }
    }
}
